services/core/shared/core/compat/common/time.js

```diff
@@ -17,9 +17,10 @@ const { getNetworkConstants } = require('./constants');
 
 let epochUnixTime;
 
+// This method is only used for SDK Version <=4
 const getEpochUnixTime = async () => {
 	const nodeConstants = await getNetworkConstants();
-	const { epoch } = nodeConstants.data ? nodeConstants.data : nodeConstants; // TODO: Fix this
+	const { epoch } = nodeConstants.data ? nodeConstants.data : nodeConstants;
```

Comment:
We could clean this up. Please verify which of the following works for SDKv5. The current implementation here was to support both SDKv4 and SDKv5:

```
const { epoch } = nodeConstants;
```
or
```
const { epoch } = nodeConstants.data;
```

Reply:
We do not get epoch for SDKv5 and I already added a comment: `This method is only used for SDK Version <= 4`.